### DISCUSSION TITLE
Parametrize book

### DISCUSSION THREAD

**Discussion by Juuso1** [quarto-dev/quarto-cli#5201] [Q&A]
Hi,

Is it possible to parametrize entire book using terminal yml params file?  

Terminal:
quarto render --profile city --cache-refresh --execute-params params.yml --to html

BR, Juuso

**Comment by mcanouil** [comment]
Yes, as you long as you defined them in the Quarto documents you are going to need them.
For example with. `knitr`:
<img width="1624" alt="image" src="https://user-images.githubusercontent.com/8896044/232335266-f8015559-54ef-4429-88c0-dbcb34043952.png">
